lib/api/manipulation.js

```diff
@@ -102,6 +102,26 @@ var uniqueSplice = function(array, spliceIdx, spliceCount, newElems, parent) {
   return array.splice.apply(array, spliceArgs);
 };
 
+exports.appendTo = function(target) {
+	if (typeof target === 'string') {
```

Comment:
This project uses two-space soft tabs. Could you change the code style here to match?